### PR TITLE
Update README and CHANGELOG for 11.0.0 stable release

### PR DESCRIPTION
**Pull request by palnabarun**
- [x] Update README
- [x] Update CHANGELOG

### CODE REVIEW

**Comment by palnabarun** [comment]
The CHANGELOG may need to be updated based on the outcome of https://github.com/kubernetes-client/python/pull/1050#issuecomment-572403936

File: README.md [hunk 1]
```diff
@@ -114,12 +114,13 @@ between client-python versions.
 | 7.0 Alpha/Beta  | Kubernetes main repo, 1.11 branch    | ✗                             |
 | 7.0             | Kubernetes main repo, 1.11 branch    | ✗                             |
 | 8.0 Alpha/Beta  | Kubernetes main repo, 1.12 branch    | ✗                             |
-| 8.0             | Kubernetes main repo, 1.12 branch    | ✓                             |
+| 8.0             | Kubernetes main repo, 1.12 branch    | ✗                             |
 | 9.0 Alpha/Beta  | Kubernetes main repo, 1.13 branch    | ✗                             |
 | 9.0             | Kubernetes main repo, 1.13 branch    | ✓                             |
 | 10.0 Alpha/Beta | Kubernetes main repo, 1.14 branch    | ✗                             |
 | 10.0            | Kubernetes main repo, 1.14 branch    | ✓                             |
 | 11.0 Alpha/Beta | Kubernetes main repo, 1.15 branch    | ✓                             |
```
Comment (micw523) [inline README.md]: When the stable version is released, the alpha/beta support reaches its end.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# v11.0.0
+
+**Deprecation Notice**
+v11.0.0 of the client follows the Kubernetes [deprecation policy](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew) and will
+be deprecated as and when Kubernetes version v1.15 gets deprecated.
```
Comment (micw523) [inline CHANGELOG.md]: I didn't quite get "as and when k8s v1.15..."?

File: README.md (hunk 1 above)
Comment (palnabarun) [reply README.md]: I think I missed that. Let me change it.

File: CHANGELOG.md (hunk 2 above)
Comment (palnabarun) [reply CHANGELOG.md]: I meant that whenever in future k8s v1.15 is deprecated, we will automatically drop support for Client version 11.0.0

File: README.md (hunk 1 above)
Comment (palnabarun) [reply README.md]: Updated.

File: CHANGELOG.md (hunk 2 above)
Comment (scottilee) [reply CHANGELOG.md]: I think it would be clearer if you removed the "as and". You can also remove "version" since you have 'v1.15'.

**Pull request merged**